### DISCUSSION TITLE
GPU and CPU Mixed Fan Curve

### DISCUSSION THREAD

**Discussion by wsm78** [Rem0o/FanControl.Releases#2079] [Q&A]
I have an RTX 4090 MSI Suprim Liquid and was wanting to use its default fan curve settings for its radiator. I want to make a mixed fan curve specifically for switching between my custom CPU coolant curve and the default GPU fan curve. Is this at all possible does anyone know?

**Comment by Rem0o** [comment]
Keep the GPU controls disabled.
Create a "sync" fan curve. Select your GPU as its source.
Create a mix, mix your sync and your CPU coolant curve.

**Reply by wsm78** [reply]
So I'm assuming there isn't a way the other way around then? If the CPU kicks up but the GPU does not?

**Reply by Rem0o** [reply]
The previously stated setup would kick up either if the gpu or CPU heats up. 

Experiment with it, you'll see for yourself how it works out!
